sycl/test-e2e/ESIMD/unified_memory_api/Inputs/gather.hpp

```diff
@@ -286,3 +286,238 @@ template <typename T, TestFeatures Features> bool testUSM(queue Q) {
   }
   return Passed;
 }
+
+template <typename T, uint16_t N, uint16_t VS, bool UseMask, bool UsePassThru,
+          bool UseProperties, typename PropertiesT>
+bool testSLM(queue Q, uint32_t MaskStride, PropertiesT) {
```

Comment:
It is funny... I don't understand what happened, but in this PR I don't see any changes for testACC(), while that function disappeared from the proposed variant of the file - now it has 2x testUSM() and 1x testSLM().

Also, my PR added testACC(): https://github.com/intel/llvm/commit/655ab100a990c83cb8576d3a84116fce2cf05ead

Reply:
Somehow `testACC` magically disappears from your variant of gather.hpp file and github does not show me that function in deleted lines.

Reply:
Sorry, my bad. Your changes do not include my yesterday fix/merge implementing gather(acc). Please do 'git merge' to pull those changes and resolve conflicts.